server/command.go

```diff
@@ -181,7 +181,7 @@ func (p *Plugin) runDisconnectCommand(user *model.User) (string, error) {
 	}
 
 	if err := p.disconnectOAuthUser(user.Id); err != nil {
-		return fmt.Sprintf("Failed to disconnect the user: %s", err.Error()), nil
+		return "Your Zoom account is not currently linked.", nil
```

Comment:
Nit pick: Instead of checking `err != nil´ an alternative approach is to check using [`errors.Is`](https://golang.org/pkg/errors/#Is). Right now, an error returned by `disconnectOAuthUser` is not printed.

Reply:
I couldn't think of a way to compare the error type but I hope the new  way of processing is acceptable.